src/tool/python/helpers.h

```diff
@@ -581,6 +581,22 @@ namespace xlang
         return count;
     }
 
+    int count_out_param(std::vector<method_signature::param_t> const& params)
```

Comment:
`std::count_if` 😉

Reply:
I changed the implementation of count_in_param/count_out_param to use count_if, but I didn't change the call site. The syntax for calling these algorithms is kinda verbose. That's getting fixed with Ranges, right?

Reply:
That's right.